pset-functions.js

```diff
@@ -91,6 +106,19 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+
+
+function wordCount(phrase){
+  let obj={};
+  phrase.split(" ").forEach(function(el,i,arr){
```

Comment:
@Ashbk32 generally, you should try to use variable names that are more descriptive, this makes your code easier to read